documentation/guide-lombok.asciidoc

```diff
@@ -0,0 +1,75 @@
+:toc: macro
+toc::[]
+
+= Lombok 
+
+https://projectlombok.org/[Lombok] is a library that works with an annotation processor and will generate code for you to save you some time and reduce the amount of boilerplate code in your project. Lombok can generate getter and setter, equals methods, automate your logging variables for your classes, and more. Follow the https://projectlombok.org/features/all[list of all the features] provided by Lombok to get an overview.
```

Comment:
Maybe you can add a hint that there is some additional setup required to use Lombok within an IDE and then link the setup page